doc/libghdl/Initialization.rst

```diff
@@ -0,0 +1,21 @@
+.. _LIB:Initialization:
+
+Initialization
+##############
+
+To be able to use `libghdl`, it first must be initialized.
+
+* Why does it must be initialized?
+* What does the initialization do?
+* Is there any use case where someone would want to use libghdl without initialization?
+* If no, then why it does not initialize itself automatically?
+* What functions need to be called during initialization? In Python bindings 3 functions are called:
+
+.. code-block:: python
+
+   libghdl.libghdl_init()
```

Comment:
This one does Ada elaboration.  Nothing visible and could be automatic (but that would make the build infrastructure more complex).